sw/host/ot_certs/src/x509/extension.rs

```diff
@@ -221,19 +221,12 @@ pub fn dice_tcb_info_extension(dice_tcb_info: &DiceTcbInfo) -> Result<X509Extens
 // the generator.
 impl template::HashAlgorithm {
     // Return the OID of the hash algorithm.
-    pub fn oid(&self) -> asn1::ObjectIdentifier {
+    pub fn objid(&self) -> asn1::ObjectIdentifier {
```

Comment:
Why's this renamed?

Reply:
FWIW, I prefer this or `obj_id` or `object_id` to `oid`, even if that is some ASN/X509 term.